src/components/HypothesisTesting/PerformTest.js

```diff
@@ -21,23 +22,60 @@ export default function PerformTest({ distType, shape, tails, mue0 }) {
     setPopArr(dataFromDistribution(shape, 2000, { mean: 69, low: 59, hi: 79 }))
   }, [shape]);
 
+/*
+  const Pop2Mean = Math.floor(Math.random() * (66 - 61 + 1)) + 61;
+
+  useEffect(() => {
+    setPopArr(dataFromDistribution(shape, 2000, { mean: Pop2Mean, low: 59, hi: 79 }))
+  }, [shape]); */
+
   const takeSample = () => {
     setSample(_.sampleSize(popArr, sampleSize));
     if (sim === 0) {
       setSim(1);
     }
   }
+  
+  let extraButton = (<> </>)
```

Comment:
I'm wondering about a second sample input. If the two samples need to be the same size, I think we can stick with the one input we already have. @ProfessorAmanda, @tbyker: can the two samples be different sizes for a two-sample hypothesis test?